config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/Bundle.java

```diff
@@ -202,4 +202,26 @@ public void setFolderTree(FolderTree folderTree) {
     public void setDependencyMap(Map<Dependency, List<Dependency>> dependencyMap) {
         this.dependencyMap = dependencyMap;
     }
+
+    public void verifyExistingFolderPath(String folderPath) {
+        boolean isValidImport = false;
+
+        // True because there isn't a specific directory.
+        if (folderPath.equals("/")) {
+            isValidImport = true;
+        }
+        
+        long matchingFolderPaths = getFolders().values()
+            .stream()
+            .filter(folder -> ("/" + folder.getPath()).equals(folderPath))
+            .count();
+        
+        if (matchingFolderPaths > 0) {
+            isValidImport = true;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define and throw a dedicated exception instead of using a generic one. ([squid:S00112](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS00112))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=131&issues=AWcZobkXkEeD8QRklwaX&open=AWcZobkXkEeD8QRklwaX)

[ISSUE_KEY]: AWcZobkXkEeD8QRklwaX